static/app/views/settings/organizationMembers/organizationMembersList.spec.tsx

```diff
@@ -595,55 +595,4 @@ describe('OrganizationMembersList', function () {
       );
     });
   });
-
-  // TODO(cathy): uncomment
```

Comment:
❓

Reply:
this test now exists here https://github.com/getsentry/sentry/pull/59666/files#diff-43f2f34cbba7732afcf9766fb1179223954ecbe69b7bc0171a027c0acded0944R204